src/pages/LoginPage.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser, faKey } from '@fortawesome/free-solid-svg-icons';
+
+import '../assets/styles/pages/LoginPage.css';
+
+function LoginPage(){
+    const [emailErrorVisibility, setEmailErrorVisibility] = useState('hidden');
+    const [passwordErrorVisibility, setPasswordErrorVisibility] = useState('hidden');
+
+    const [emailErrorMessage, setEmailErrorMessage] = useState('');
+    const [passwordErrorMessage, setPasswordErrorMessage] = useState('');
+
+    const [currentEmail, setCurrentEmail] = useState('');
+    const [currentPassword, setCurrentPassword] = useState('');
+
+    const handleClick = e => {
+        if(e.target.outerText === 'Sign Up') {
+            /////////////////////////////////
+            //REDIRECT TO SIGN UP PAGE HERE//
+            /////////////////////////////////
+            console.log('REDIRECT TO SIGN UP PAGE')
+            return 0;
+        }
+        e.preventDefault();
+
+        setEmailErrorMessage('');
+        setPasswordErrorMessage('');
+        setEmailErrorVisibility('hidden');
+        setPasswordErrorVisibility('hidden');
+
+        // Validations
+
+        let emailIsValid = true;
+        let passwordIsValid = true;
+
+        let emailHasAnAt = false;
+        let emailHasADomain = true;
+        currentEmail.split('').forEach((char, index, arr) => {
```

Comment:
You can separate validation functionality from this function, call it validateInputs or validation, and pass arguments to update hooks if there some errors.

Reply:
Done, I separated the validation from the handleSubmit